### DISCUSSION TITLE
is there a way for it to start in the top left?

### DISCUSSION THREAD

**Discussion by aidanhadley** [flxzt/rnote#803] [Q&A]
similar to onenote, is there a setting for the note to start in the top left and expand to the right and down? so far, i'm absolutely loving this app!!

**Comment by flxzt** [comment]
thanks to @D0V4HKIIN this is already implemented, called "half-infinite" layout. you can select it through the canvas menu (the one that can be opened through the button with the document symbol in the headerbar). When you click the "document-layout" item the different layouts can be selected.
